vllm/compilation/backends.py

```diff
@@ -263,8 +322,14 @@ class VllmBackend:
     returned_callable: Callable
 
     def __init__(self, ):
-        # every instance of VllmBackend has its own graph pool
-        self.graph_pool = torch.cuda.graph_pool_handle()
+        global global_graph_pool
+        if global_graph_pool is None:
+            global_graph_pool = torch.cuda.graph_pool_handle()
```

Comment:
why does the graph_pool need to be a global?

Reply:
this way multiple models can share the same graph pool (to save memory). for example, the vision tower and the language tower of the same vision-language model.